### PR TITLE
Angular 5 changes and some small typo corrections

### PR DESCRIPTION
**Pull request by nioe**
* Removed deprecated `Http` in favour of the new `HttpClient`
* Corrected some wording, typos and formatting issues
* TODO: Change test examples because `MockBackend` is deprecated (added TODO in code)

An example of the new InMemoryWebAPI can be found in my example project on Branch 12_Testing_4. ;)

### CODE REVIEW

File: slides/06_Templates_Grundlagen.html [hunk 1]
```diff
@@ -257,10 +262,10 @@ <h3>Beispiele</h3>
                          <button (click)="createPerson()">Neue Person</button>
                      </code>
                     <code class="hljs html" data-trim contenteditable>
-                         &ltaddress (customEvent)="machWas()"></address>
+                         <postal-address (customEvent)="machWas()"></postal-address>
                      </code>
                     <code class="hljs html" data-trim contenteditable>
-                         &ltpeople (deletePerson)="delete($event)"></people>
+                         <people (deletePerson)="delete($event)"></people>
```
Comment (joachimprinzbach) [inline slides/06_Templates_Grundlagen.html]: Do stick with the &lt, otherwise the code wont be rendered correctly

**Review by joachimprinzbach — CHANGES_REQUESTED** [review]
Thanks 👍

File: slides/06_Templates_Grundlagen.html (hunk 1 above)
Comment (nioe) [reply slides/06_Templates_Grundlagen.html]: My bad... Overlooked it while I was merging my changes... Will change it back. ;)

**Comment by joachimprinzbach** [comment]
closes #26

**Review by joachimprinzbach — APPROVED** [review]

**Pull request merged**